src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -33,7 +33,7 @@ instance has_mul {G : Type*} [has_mul G] [topological_space G] [charted_space H'
 ⟨λ f g, ⟨f * g, f.smooth.mul g.smooth⟩⟩
 
 @[simp, to_additive]
-lemma coe_mul {G : Type*} [has_mul G] [topological_space G] [charted_space H' G]
+lemma mul_coe {G : Type*} [has_mul G] [topological_space G] [charted_space H' G]
```

Comment:
The old names are the ones we usually use here. `continuous_map` is the odd one out.

Reply:
Ok

Reply:
There is a lot of places in the library where `mul_coe` is there... Do you think you could make a refactor changing it only where it needs to be changed? Or can you explain the logic to choose when it needs to be `coe_mul` and when it has to be `mul_coe`?